editor/editor_inspector.cpp

```diff
@@ -1037,6 +1124,10 @@ EditorProperty::EditorProperty() {
 	bottom_editor = nullptr;
 	menu = nullptr;
 	set_process_shortcut_input(true);
+
+	warning_dialog = memnew(AcceptDialog);
```

Comment:
This should be created only on demand.

Reply:
I took this code from `SceneTreeEditor`, so I will also fix it there then.

Reply:
Ah right, I see what you mean now. I misremembered, thinking the dialog was in the inspector and not per property for some reason.